easybuild/easyconfigs/m/METIS/METIS-5.1.0-use-doubles.patch

```diff
@@ -0,0 +1,13 @@
+# Use double for floating point (64 bit) in METIS by default
```

Comment:
@wpoely86 add author please, and maybe also mention *why* you're making this chance (required by OpenFOAM 4.0?)